### PR TITLE
[node] Add pending var on node/net.d.ts

### PR DESCRIPTION
**Pull request by tolbon**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

Hello Types Team,
I add pending var on @types/node on net.d.ts file
You can find here [Nodejs Doc](
https://nodejs.org/dist/latest-v19.x/docs/api/net.html#socketpending)

### CODE REVIEW

**Review by tolbon — COMMENTED** [review]
New push to respect JsDoc Format

**Comment by Semigradsky** [comment]
@tolbon Could you please also add to these folders?
- `/ts4.8/*`
- `/v16/*`
- `/v16/ts4.8/*`
- `/v14/*`
- `/v14/ts4.8/*`

File: types/node/net.d.ts [hunk 1]
```diff
@@ -261,6 +261,13 @@ declare module 'net' {
          * @since v6.1.0
          */
         readonly connecting: boolean;
+
```
Comment (Semigradsky) [inline types/node/net.d.ts:264]: There are trailing whitespaces here

File: types/node/net.d.ts [hunk 2]
```diff
@@ -261,6 +261,13 @@ declare module 'net' {
          * @since v6.1.0
          */
         readonly connecting: boolean;
+        
+        /**
+         * This is `true` if the socket is not connected yet, either because `.connect()`
```
Comment (Semigradsky) [inline types/node/net.d.ts:266]: There is trailing whitespace here, see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63814/checks

**Comment by tolbon** [comment]
Hello @Semigradsky 

- [ ] I remove trailing whitespace

- [ ] Add in these folders (like you ask) :
    /ts4.8/*
    /v16/*
    /v16/ts4.8/*
    /v14/*
    /v14/ts4.8/*

- [ ] and i permit to respect property Alphabetic order

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@tolbon thanks!

**Comment by tolbon** [comment]
Reopen

**Pull request closed without merging**